src/time-picker/time-picker.scss

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../menu/index';
+
+@mixin iui-time-picker {
+  @include iui-reset;
+  @include iui-font-family;
```

Comment:
Maybe should use `iui-dropdown` mixin here to be consistent

Reply:
Should not.. it adds shadow, which brakes things a little..